server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -73,6 +68,7 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
     public static final String REQUEST_EXTENSION_CLUSTER_SETTINGS = "internal:discovery/clustersettings";
     public static final String REQUEST_OPENSEARCH_NAMED_WRITEABLE_REGISTRY = "internal:discovery/namedwriteableregistry";
     public static final String REQUEST_OPENSEARCH_PARSE_NAMED_WRITEABLE = "internal:discovery/parsenamedwriteable";
+    public static final String REQUEST_ACTION_LISTENER_ON_FAILURE = "internal:discovery/actionlisteneronfailure";
```

Comment:
Small change here, since all request action names from the extension to OpenSearch begin with `REQUEST_EXTENSION_` we should follow this pattern. Perhaps change to `REQUEST_EXTENSION_ACTION_LISTENER_ON_FAILURE`